### DISCUSSION TITLE
Help cant find way to view side bar

### DISCUSSION THREAD

**Discussion by StephenWong-GIT** [Sequel-Ace/Sequel-Ace#1918] [Q&A] [answered]
after updating today, i no longer can find or view the sidebar that held all my tables in my db, i have to now go through the navigator which is seriously an inconvenience. Ive tried looking through the shortcuts but nothing says it can bring up the sidebar navigation that used to be there pre update. This whole new UI.. kinda gross ngl

**Comment by StephenWong-GIT** [comment]
Also why is stability on mac so bad? Like truly, i  cent even open a new db or refresh without it crashing, its been crashing literally everyday non stop, to the point id rather do all my table viewing through phpmyadmin. this is disgusting

**Comment by StephenWong-GIT** [comment, marked as answer]
okay i fixed it by clicking a billion times to try and drag out the sidebar. please whatever update you do in the future never have a side 0 or at least have an option to show/hide sidebar in the view tab its currently not there as of update 11/09/2023. This is a fucking horrible oversight and if i didnt need to use your program i wouldnt.

**Reply by StephenWong-GIT** [reply]
and if you do that suggestion, when you show/hide the show always opens at default width. boom done. issue for a billion people using this dog water app fixed.

**Reply by sav4n** [reply]
I faced this same issue today. 
**Thanks You saved me**

**Reply by cAstraea** [reply]
ohhh it was there all along ;D even in my latest version. I tried dragging it and there it was. dunno why it disappeared

**Reply by stychos** [reply]
> okay i fixed it by clicking a billion times to try and drag out the sidebar. please whatever update you do in the future never have a side 0 or at least have an option to show/hide sidebar in the view tab its currently not there as of update 11/09/2023. This is a fucking horrible oversight and if i didnt need to use your program i wouldnt.

You have a variety of another things to try, DBeaver, Workbench etc. No need to blame developers of the truly free product for some glitches and crashes. Also, those glitches and crashes are the legacy of the predecessor, which isn't maintainable at all nowadays.

**Reply by aleph-naught2tog** [reply]
2025 here checking in and saying THANK YOU for this!

**Reply by JoeFRUS** [reply]
Hooooly shit I just lost an hour of my life to this, thank you!
